charts/fleet/charts/gitjob/values.yaml

```diff
@@ -9,3 +9,9 @@ tekton:
 global:
   cattle:
     systemDefaultRegistry: ""
+
+# http[s] proxy server
+# proxy: http://<username>@<password>:<url>:<port>
+
+# comma separated list of domains or ip addresses that will not use the proxy
+noProxy: 127.0.0.0/8,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16
```

Comment:
Probably not really required for gitjob, but usually, it makes sense to include `10.43.0.1` (local access to kubernetes api) to `NO_PROXY`

Reply:
> Probably not really required for gitjob, but usually, it makes sense to include `10.43.0.1` (local access to kubernetes api) to `NO_PROXY`

Thanks @ytsarev, nice catch. I guess I should also add `.svc` and `.cluster.local`, according to https://github.com/rancher/rancher/pull/29491, will amend.

Reply:
@ytsarev, It looks like [Service Cluster CIDR is configurable](https://github.com/rancher/docs/blob/master/content/k3s/latest/en/installation/install-options/server-config/_index.md#networking), so it's better to leave it for user to override.
However I've added `.svc` and `.cluster.local` domains.

Reply:
Perfect :+1: